### PR TITLE
Enabling Pages on a private repo requires a paid account

### LINKED ISSUE #26091: Recipes: Enabling Pages on a private repo requires a paid account
### Describe your suggestion

You offer the steps to set up pages, but if your repo is private it requires an account upgrade. It is worth mentioning this for people who want to keep their repo private. You do mention there are multiple ways, but adding a note about what the other ways are is helpful.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-recipes

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### (Optional) Discord Name

_No response_

### (Optional) Additional Comments

Thank you for this wonderful resource, I am utilizing it to teach my niece to code! <3

### PR DESCRIPTION
**Pull request by Sushant9473**
## Because

To provide  a note that  "Enabling Pages on a private repo requires a paid account"

## This PR

- A note displaying "Enabling Pages on a private repo requires a paid account"

## Issue

Closes #26091

## Additional Information

Lesson Url
https://www.theodinproject.com/lessons/foundations-recipes

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Review by CouchofTomato — CHANGES_REQUESTED** [review]
Can we change the wording to reflect what you wrote in the PR

"Enabling Pages on a private repo requires a paid account"

File: foundations/html_css/flexbox/project-landing-page.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ If you want to show your work (the project) to others, or submit a solution belo
 
 GitHub allows you to publish web projects directly from a GitHub repository. Doing this will allow you to access your project from `your-github-username.github.io/your-github-repo-name`
 
+Note- Enabling pages on private repo requires paid account.
```
Comment (KevinMulhern) [inline foundations/html_css/flexbox/project-landing-page.md:63]: Can we use [one of our note boxes](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#note-boxes) for this please?

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

File: foundations/html_css/flexbox/project-landing-page.md (hunk 1 above)
Comment (Sushant9473) [reply foundations/html_css/flexbox/project-landing-page.md:63]: > Can we use [one of our note boxes](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#note-boxes) for this please?

Thanks for informing . i will add the inbuilt note feature.

**Comment by Sushant9473** [comment]
> Can we change the wording to reflect what you wrote in the PR
> 
> "Enabling Pages on a private repo requires a paid account"



> Can we change the wording to reflect what you wrote in the PR
> 
> "Enabling Pages on a private repo requires a paid account"

worded PR correctly

**Comment by Sushant9473** [comment]
> ## Because
> To provide a note that "Enabling Pages on a private repo requires a paid account"
> 
> ## This PR
> * A note displaying "Enabling Pages on a private repo requires a paid account"
> 
> ## Issue
> Closes #26091
> 
> ## Additional Information
> Lesson Url https://www.theodinproject.com/lessons/foundations-recipes
> 
> ## Pull Request Requirements
> * [x]  I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
> * [x]  The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
> * [x]  The `Because` section summarizes the reason for this PR
> * [x]  The `This PR` section has a bullet point list describing the changes in this PR
> * [x]  If this PR addresses an open issue, it is linked in the `Issue` section
> * [x]  If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
> * [x]  If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

**Comment by Sushant9473** [comment]
I am closing this pull request as i am not able to commit further to this branch because my lack of knowledge of working with git .  I will open another pull request resolving all the issue mentioned here

**Pull request closed without merging**